content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+of the most common concerns:
+
+- [Network topologies](/docs/ops/deployment/deployment-models#network-models):
+  one or two networks
+
+- [Control plane topologies](/docs/ops/deployment/deployment-models#control-plane-models):
+  multi-primary, primary-remote
+
+Before you begin, review the [deployment models guide](/docs/ops/deployment/deployment-models)
+which describes the foundational concepts used throughout this guide.
+
+## Requirements
+
+This guide requires that you have two Kubernetes clusters with any of the
+[supported versions](/docs/setup/platform-setup).
+
+If the clusters are on different networks (i.e. no direct connectivity between
+pods on different clusters), they must have ingress gateway services which are
+accessible from the other cluster, ideally using L4 network load balancers
+(NLBs). Not all cloud providers support NLBs and some require special
+annotations to use them, so please consult your cloud provider’s documentation
+for enabling NLBs for service object type load balancers. When deploying on
+platforms without NLB support, it may be necessary to modify the health checks
+for the load balancer to register the ingress gateway.
+
+## Environment Variables
+
+This guide will refer to two clusters named `FRED` and `BARNEY`. The following
+environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_FRED` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `FRED` cluster.
+`CTX_BARNEY` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `BARNEY` cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_FRED=cluster-fred
+$ export CTX_BARNEY=cluster-barney
+{{< /text >}}
+
+## Configure Trust
+
+A multicluster service mesh deployment requires that you establish trust
+between all clusters in the mesh. Depending on the requirements for your
+system, there may be multiple options available for establishing trust.
+See [certificate management](/docs/tasks/security/cert-management/) for
+detailed descriptions and instructions for all available options.
+Depending on which option you choose, the installation instructions for
+Istio may change slightly.
+
+This guide will assume that you use a common root to generate intermediate
+certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
+to generate and push a secret to both the `FRED` and `BARNEY` clusters.
+
+{{< tip >}}
+If you currently have a single cluster with a self-signed CA (as described
+in [Getting Started](/docs/setup/getting-started/)), you’ll need to
+[change the CA](/docs/tasks/security/cert-management/) to support multiple
```

Comment:
That link isn't clear, what do they actually need to do if they already have Istio installed with a self-signed cert? Can we point them at explicit instructions on how to migrate from self-signed to intermediate?